### PR TITLE
Improvements over Blaze i3

### LINKED ISSUE #12539: Blaze i3 Quick Enhancements
Clone of Android issue - https://github.com/woocommerce/woocommerce-android/issues/11321

After the user flow review session, a lot of good feedback came up. Full context here: pe5sF9-2Bb-p2

This issue summarizes a list of small changes that can be applied to the flow and that might have a significant impact. 
Bigger effort changes have been added to other GH issues and will be deferred to Blaze i4 Project. 

The small task added to this GHG issue can be tackled bit by bit while we work on other projects. We don't plan to allocate specific team effort for these as they should be 1h-2h changes at most. 

```[tasklist]
### Quick Enhancements
- [x] Intro screen small changes: Remove purple flames, explain what the $5 USD budget is (daily,weekly,total, etc)
- [x] Preview Screen: Add a new copy below the budget section: `Audience`.
- [x] Preview Screen: Add a label on the ad preview specifying if the used content for the ad was generated by AI. Hide this label as soon as the user edits the content manually
- [x] Small redesign to the Blaze view inside My Store tab: Thd2rdNU8zSaLkyKZGsOSn-fi-77_29558 This will be tackled as part of the dynamic dashboard project
- [x] Align Blaze statuses in all platforms. Both iOS and Android should have the exact same statuses as the web displays.
```

### PR DESCRIPTION
**Pull request by pmusolino**
Closes: #12539

## Description
Full context about these changes here pe5sF9-2Bb-p2

List of changes:
- Removed purple flames in the intro screen and explain that the $5 USD budget is daily into the Blaze Intro Screen.
- In Blaze Preview Screen added a new text below the budget section, called `Audience`.
- Updated 2 statuses for Blaze campaigns where the returned value from the API differs from what we display in the UI. `pending` -> `In Moderation` & `finished` -> `Completed`
- In Blaze Preview Screen, added a label on the ad preview specifying if the used content for the ad was generated by AI. The label is hidden as soon as the user edits the content manually + added some unit tests

## Testing information
Launch Blaze campaign creation flow and review that each one of the tasks is applied correctly.

---
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">View files have been modified, but no screenshot or video is included in the pull request. Consider adding some for clarity.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/woocommerce/woocommerce-ios/milestone/211">19.3</a>. This milestone is due in less than 2 days.<br />
Please make sure to get it merged by then or assign it to a milestone with a later deadline.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WooCommerce/Classes/Extensions/BlazeCampaignListItem+Customizations.swift [hunk 1]
```diff
@@ -53,12 +53,12 @@ extension BlazeCampaignListItem.Status {
     }
 
     private enum Localization {
-        static let pending = NSLocalizedString("Pending", comment: "Status name of an active Blaze campaign")
+        static let inModeration = NSLocalizedString("In Moderation", comment: "Status name of a pending Blaze campaign")
```
Comment (itsmeichigo) [inline WooCommerce/Classes/Extensions/BlazeCampaignListItem+Customizations.swift]: Nit: Please follow the new rule of separate keys and values for localized strings p91TBi-aJl-p2. I believe it's worth it to update other strings in the enum as well.

**Review by itsmeichigo — COMMENTED** [review]
Thank you Paolo for handling the improvements!

Next time, can you add before/after screenshots in the PR description to make it easier to follow? I believe that's one of the best practices for PRs.

Regarding the "Suggested by AI" text, the text is invisible in dark mode now:



WDYT about moving this label under the description to clarify that the content was generated, not the image? The layout also looks nicer that way.

**Comment by pmusolino** [comment]
Thank you @itsmeichigo for the review!

I followed your suggestion, and moved the label "Generated with AI" under the description, I agree it's more clear in this way. Since the background it's always the same, there are no changes to be made on light/dark mode.
046f633dc915eeae208fdc81ddf45e5c3999446f

File: WooCommerce/Classes/Extensions/BlazeCampaignListItem+Customizations.swift (hunk 1 above)
Comment (pmusolino) [reply WooCommerce/Classes/Extensions/BlazeCampaignListItem+Customizations.swift]: Done here 5a27ac1f140817a0cc4a94c4749852f87609344f I updated also the other `NSLocalizedString` in this file.

**Review by itsmeichigo — APPROVED** [review]
Thanks for the updates Paolo!

**Pull request merged**